libs/sr/solver/src/parentWorker.ts

```diff
@@ -216,3 +228,18 @@ function convertRelicToStats(relic: ICachedRelic): RelicStats {
     },
   }
 }
+
+function convertLightConeToStats(lightCone: ICachedLightCone): LightConeStats {
+  const calc = srCalculatorWithEntries(lightConeData(lightCone))
+
+  return {
+    id: lightCone.id,
+    stats: {
+      ...objKeyMap(
+        ['hp', 'atk', 'def'] as const,
+        (stat) => calc.compute(own.base[stat].with('sheet', 'lightCone')).val
+      ),
+      [lightCone.key]: 1,
```

Comment:
@nguyentvan7 Place for a lightcone key accumulator?
https://github.com/frzyc/genshin-optimizer/pull/2531/files#r1831295259